### ISSUE TITLE
evaluate

### ISSUE THREAD

**Issue by Attrayy** [brepdiff/brepdiff#10]
I would like to ask how you evaluate whether the model training is good or not. Is it based on the validation loss (val loss)

<img width="343" height="32" alt="Image" src="https://github.com/user-attachments/assets/d6324686-53ed-49bd-906b-e3c3e7b59211" />
